dashboard/test/models/contact_rollups_raw_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class ContactRollupsRawTest < ActiveSupport::TestCase
+  test 'extract_email_preferences adds all email preferences' do
+    email_preference = create :email_preference
+    ContactRollupsRaw.extract_email_preferences
+
+    expected_data = {opt_in: email_preference.opt_in}
+    result = ContactRollupsRaw.find_by(
+      email: email_preference.email,
+      sources: "dashboard.#{email_preference.class.table_name}"
+    )
+
+    assert_equal expected_data, result.data.symbolize_keys
+  end
+
+  test 'bunch of email prefences works' do
```

Comment:
How about something like `extract_email_preferences gets all rows` for this test and `extract_email_preferences gets right column` for the other? It will be easier to find all the tests for a certain function once we add more extraction functions.

Reply:
I noticed I forgot to update these test names but didn't push the commit, just added :)

Reply:
My updates didn't quite get to your naming suggestion though, will change!

Reply:
Sounds good :)